### PR TITLE
VIA changes

### PR DESCRIPTION
**Pull request by MarkY-LunarG**
Commit Joey's VIA changes to look in the HKEY_CURRENT_USER registry location for loader registry settings.  Plus, a few cleanups by me.

### CODE REVIEW

**Comment by mark-lunarg** [comment]
The source needs to be run through clang-format.

**Comment by mark-lunarg** [comment]
OTT, looks good to me.

File: via/via.cpp [hunk 1]
```diff
@@ -1773,6 +1783,34 @@ ErrorResults PrintDriverInfo(void) {
...
+void PrintUninstallRegInfo(HKEY reg_folder, char *output_string, char *count_string, char *generic_string, char *version_string, unsigned int& install_count) {
+    uint32_t i = 0;
+    // Find all Vulkan Runtime keys in the registry, and loop through each.
+    while (FindNextRegKey(reg_folder, g_uninstall_reg_path, "VulkanRT", i, MAX_STRING_LENGTH - 1, output_string)) {
+        snprintf(count_string, MAX_STRING_LENGTH - 1, "[%d]", i++);
+
+        snprintf(generic_string, MAX_STRING_LENGTH - 1, "%s\\%s", g_uninstall_reg_path, output_string);
+
+        // Get the version from the registry
+        if (ReadRegKeyString(reg_folder, generic_string, "DisplayVersion", MAX_STRING_LENGTH - 1, version_string)) {
```
Comment (ardaLunarG) [inline via/via.cpp]: Empty if ?

File: via/via.cpp [hunk 2]
```diff
@@ -1906,6 +1922,80 @@ ErrorResults PrintRunTimeInfo(void) {
...
 }
 
+bool PrintSdkUninstallRegInfo(HKEY reg_folder, char *output_string, char *count_string, char *generic_string) {
+    uint32_t i = 0;
+    bool found = false;
+    while (FindNextRegKey(reg_folder, g_uninstall_reg_path, "VulkanSDK", i, MAX_STRING_LENGTH, output_string)) {
+        found = true;
+        snprintf(count_string, MAX_STRING_LENGTH - 1, "[%d]", i++);
+        snprintf(generic_string, MAX_STRING_LENGTH - 1, "%s\\%s", g_uninstall_reg_path, output_string);
+        if (ReadRegKeyString(reg_folder, generic_string, "InstallDir", MAX_STRING_LENGTH, output_string)) {
```
Comment (ardaLunarG) [inline via/via.cpp]: One more Empty if

File: via/via.cpp (hunk 1 above)
Comment (MarkY-LunarG) [inline via/via.cpp]: Fixed

File: via/via.cpp (hunk 2 above)
Comment (MarkY-LunarG) [inline via/via.cpp]: Fixed.

**Pull request merged**